app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -194,6 +196,55 @@ class MainActivity : AppCompatActivity(), WebViewController {
         if (connectionHelper.connected) webView.loadUrl(url)
     }
 
+    @Suppress("DEPRECATION")
+    private fun initLocale() {
+        runOnUiThread {
+            webView.evaluateJavascript("window.localStorage.getItem('${apiClient.currentUserId}-language')") { result ->
+                if (!setLocale(result.unescapeJson(), "userSettings")) {
+                    GlobalScope.launch {
```

Comment:
Don't use GlobalScope, it should only be used in very rare circumstances.
```suggestion
                    lifecycleScope.launch {
```